relayer/chains/substrate/errors.go

```diff
@@ -0,0 +1,8 @@
+package substrate
+
+const (
+	ErrTextConsensusStateNotFound                   = "consensus state not found: %s"
+	ErrTextSubstrateDoesNotHaveQueryForTransactions = "substrate chains do not support transaction querying"
+	ErrBeefyAttributesNotFound                      = "beefy authorities not found: storage key %v, paraids %v, block hash %v"
+	ErrBeefyConstructNotFound                       = "beefy construct not found: storage key %v, authorities %v, block hash %v"
+)
```

Comment:
Error messages are typically plain strings which can later be wrapped with complimentary data at the call site where the error is observed/returned.

I would drop the formatting verbs e.g. `%v` and then wherever these errors are returned wrap them using a call to `fmt.Errorf`

e.g. 

```
const (
	ErrTextConsensusStateNotFound                   = "consensus state not found"
	ErrTextSubstrateDoesNotHaveQueryForTransactions = "substrate chains do not support transaction querying"
	ErrBeefyAttributesNotFound                      = "beefy authorities not found"
	ErrBeefyConstructNotFound                       = "beefy construct not found"
)

(inside some function where an error is observed)

val, err := sp.Foo()
if err != nil {
  return fmt.Errorf("%s: storage key %v, paraIDs %v, block hash %v", ErrBeefyAttributesNotFound, key, ids, hash)
}


```